portfolio/src/main/webapp/script.js

```diff
@@ -168,7 +210,6 @@ function removeAllComments(allComments)
 /** Creates an <ul> element containing text */
 function createListElement(text)
 {
-
```

Comment:
Why function name has word List inside?

Reply:
This function creates an unordered list element to have a container for the comments in HTML.